iOS/Controllers/MapViewController.cs

```diff
@@ -567,23 +572,29 @@ private void MapView_GeoViewDoubleTapped(object sender, GeoViewInputEventArgs e)
                         var roomMarker = new PictureMarkerSymbol(new RuntimeImage(mapPin));
                         roomMarker.OffsetY = uiImagePin.Size.Height * 0.65;
 
-                        // Create graphic
-                        var mapPinGraphic = new Graphic(GeometryEngine.LabelPoint(idResults.GeoElements.First().Geometry as Polygon), roomMarker);
+                        // Create graphic
+                        var mapPinGraphic = new Graphic(idResults.GeoElements.FirstOrDefault().Geometry.Extent.GetCenter(), roomMarker);

```

Comment:
`idResults` can be empty, right? If no feature was hit during identify or layer is not FeatureLayer?